funding/manager_test.go

```diff
@@ -4493,3 +4493,62 @@ func TestCommitmentTypeFundmaxSanityCheck(t *testing.T) {
 		}
 	}
 }
+
+// TestFundingManagerNoEchoChanType tests that the funding flow is aborted if
+// the peer fails to echo back the channel type in AcceptChannel.
+func TestFundingManagerNoEchoChanType(t *testing.T) {
```

Comment:
structure nittynit: nice to have the test in a commit before the bug fix triggering the condition and then updated in the fix commit to show it's addressed.